services/QuillLMS/app/models/change_log.rb

```diff
@@ -110,4 +142,38 @@ def record_is_not_being_created_from_cms?
   def needs_a_changed_record_id?
     applies_to_single_record? && record_is_not_being_created_from_cms?
   end
+
+  def serializable_hash(options = nil)
+    options ||= {}
+
+    super(options.reverse_merge(
+      only: [:id, :action, :changed_attribute, :changed_record_type, :changed_record_id,
+             :explanation, :new_value, :previous_value, :created_at, :updated_at, :user_id],
+      methods: [:comprehension_action, :comprehension_url, :comprehension_name, :conjunctions, :user, :updated_local_time]
+    ))
+  end
+
+  def comprehension_action
+    "#{changed_record&.change_log_name} - #{action}" if changed_record&.respond_to? :change_log_name
+  end
+
+  def comprehension_url
+    changed_record&.url if changed_record&.respond_to? :url
+  end
+
+  def comprehension_name
```

Comment:
I don't have a good suggestion for a name, but again, let's name it something with `comprehension`.